app/actions/remote/channel.ts

```diff
@@ -459,3 +459,57 @@ export const switchToChannelByName = async (serverUrl: string, channelName: stri
         return {error};
     }
 };
+
+export const fetchChannels = async (serverUrl: string, teamId: string, page = 0, perPage: number = General.CHANNELS_CHUNK_SIZE) => {
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        const channels = await client.getChannels(teamId, page, perPage);
```

Comment:
So far, this function is to get "all the channels in the team". Since we won't receive websocket events about them, we probably should keep them only in memory, to avoid stale and outdated data.

Reply:
I guess we can leave it like this and in the event we need to store them we can pass an optional `fetchOnly` argument and augment this function